setup.py

```diff
@@ -40,33 +40,33 @@
 
     python_requires='~=3.6',
     install_requires=[
-        'django>=1.11.27,<2',
+        'django>=2.2.12,<3',
         'boto3>=1.7',
         'cobalt>=3.1',
         'django-ckeditor>=5.8',
         'dj-database-url>=0.3.0',
         'django-activity-stream>=0.7.0',
-        'django-allauth>=0.36.0,<0.41.0',  # 0.41.0 moved to django 2
+        'django-allauth==0.41.0',
```

Comment:
As a library vendor, it's better to be flexible in the libraries that you support (see https://stackoverflow.com/a/44938662). This means that consumers of your library have flexibility as well. For example, if `0.42.0` came out, a consumer wouldn't be able to upgrade because we have pinned our dependency.

So, this should change to `>=0.41.0` (ditto for other versions that are pinned with `==`).

The flip side of this is in your end application that pulls in a set of libraries, they should be pinned as far as possible, so that you don't get surprise minor version upgrades during your deployments. But that's a different issue :)

Reply:
I had to pin some of the packages to identify which versions were bringing in breaking changes.  Let me pin the ones that are stable